index.js

```diff
@@ -629,6 +629,18 @@ const publicApi = {
         return this;
     },
 
+    /**
+     * If enabled, the Aurelia plugin is enabled
```

Comment:
Can you add a link to Aurelia here? And what are some valid callback options. We usually like to show an example :)

Reply:
Done :)